src/components/FileList/FileList.js

```diff
@@ -235,6 +237,16 @@ export default class FileList extends PureComponent {
     onChange(`described.facets.${facet}`, newGlobs)
   }
 
+  onLicenseChange(record, license) {
```

Comment:
nit: You can get some perf increases for free by binding this method here rather than inside of render on ln: 310. 

Also, FWIW, most, if not all of the methods of this class are bound in the method definition using arrow functions instead of binding in render.

Reply:
Agreed.  Using arrow function is more readable, e.g. `onLinceseChange = (record, license) =>{}`

Inside renderLicneseCell on line 310, however, parameter `record` still needs to be passed in.  Looks like two options:

 `onChange={license => this.onLicenseChange(record, license)}`

or

 `onChange={this.onLicenseChange.bind(null, record)}`

So binding in the render is still necessary.